plugins/systems/input/Input.hpp

```diff
@@ -54,6 +55,8 @@ namespace Systems {
                 auto &reg = engine.getRegistry();
                 ECS::Entity projectile = reg.createEntity();
 
+                std::cerr << "New entity for projectile" << projectile << std::endl;
```

Comment:
debug print or usefull print ?

Reply:
It is useful to keep track of the IDs used for creating projectile, to see how loaded the vectors can get